### PR TITLE
Updated generate doc to include aliases

### LINKED ISSUE #12200: Document all the possible cli command abbreviations
### Bug Report or Feature Request (mark with an `x`)
```
- [x] bug report -> please search issues before submitting
- [ ] feature request
```

### Command (mark with an `x`)
```
- [ ] new
- [ ] build
- [ ] serve
- [ ] test
- [ ] e2e
- [ ] generate
- [ ] add
- [ ] update
- [ ] lint
- [ ] xi18n
- [ ] run
- [ ] config
- [ ] help
- [ ] version
- [x] doc
```

### Desired functionality
I've seen examples of command abbreviations like `ng g c <component-name>` to generate components and `ng g cl <class-name>` to generate classes, but these are undocmented.

All the the possible abbreviations ought to be documented.

### PR DESCRIPTION
**Pull request by adampenn**
Fixes #12200 

Added alias section to each doc for the generate commands.

Added appShell back to generate as it seemed to be missing.

### CODE REVIEW

**Comment by adampenn** [comment]
CLA Should be signed

File: docs/documentation/generate.md [hunk 1]
```diff
@@ -5,20 +5,24 @@
...
- - [class](generate/class)
- - [component](generate/component)
- - [directive](generate/directive)
- - [enum](generate/enum)
- - [guard](generate/guard)
- - [interface](generate/interface)
- - [module](generate/module)
- - [pipe](generate/pipe)
- - [service](generate/service)
+ - [class](generate/class.md)
```
Comment (clydin) [inline docs/documentation/generate.md]: Please see the comment at the top of this file.  The `.md` extension should not be present.

Comment (adampenn) [reply docs/documentation/generate.md]: I can remove them but that means the links dont work in git hub.

Comment (clydin) [reply docs/documentation/generate.md]: Yes, that is correct.  The files are published to the wiki on release.  They are not intended to be used directly from the repository.

Please squash and change the commit message type to `docs`.

Thank you.

Comment (adampenn) [reply docs/documentation/generate.md]: Im sorry what do you mean by Please squash and change the commit message type to docs

Comment (clydin) [reply docs/documentation/generate.md]: There should only be one commit with a message starting with `docs(@angular/cli):`.
If you prefer, the caretaker can handle it when they merge.

However, for information on rebasing please see this guide: https://help.github.com/articles/about-git-rebase/

Comment (adampenn) [reply docs/documentation/generate.md]: Done. I am familiar with rebasing just hadn't done a squash before so didn't know that was an option. Thanks for the info.

**Comment by adampenn** [comment]
@clydin The commit is updated and it is passing all checks. Please let me know if there is anything else you would like for me to do.

**Pull request merged**